src/test/java/com/lmax/disruptor/dsl/DisruptorTest.java

```diff
@@ -86,6 +86,21 @@ public void tearDown() throws Exception
         executor.joinAllThreads();
     }
 
+    @Test
+    public void shouldHaveStartedAfterStartCalled() throws Exception {
+      if (disruptor.hasStarted())
+      {
+        fail("Should only be set to started after start is called");
+      }
+
+      disruptor.start();
+
+      if (!disruptor.hasStarted())
+      {
+        fail("Should be set to started after start is called");
+      }
+    }
```

Comment:
This test uses 2-space indentation while all other code in the codebase, and surrounding tests, use 4-space indentation

Reply:
sorry will fix. I'm mostly a scala developer so 2 space has been ingrained into my thumbs!